apps/server/src/modules/files-storage/entity/filerecord.entity.spec.ts

```diff
@@ -158,4 +158,245 @@ describe('FileRecord Entity', () => {
 			});
 		});
 	});
+
+	describe('hasSameName is called', () => {
+		describe('WHEN name is equal', () => {
+			const setup = () => {
+				const name = 'name123';
+				const fileRecord = fileRecordFactory.build({ name });
+
+				return { fileRecord, name };
+			};
+
+			it('should be true', () => {
+				const { fileRecord, name } = setup();
+
+				const result = fileRecord.hasSameName(name);
+
+				expect(result).toBe(true);
+			});
+		});
+
+		describe('WHEN name is not equal', () => {
+			const setup = () => {
+				const name = 'name123';
+				const fileRecord = fileRecordFactory.build({ name: 'name' });
+
+				return { fileRecord, name };
+			};
+
+			it('should be false', () => {
+				const { fileRecord, name } = setup();
+
+				const result = fileRecord.hasSameName(name);
+
+				expect(result).toBe(false);
+			});
+		});
+	});
+
+	describe('getName is called', () => {
+		describe('WHEN name exists', () => {
+			const setup = () => {
+				const name = 'name123';
+				const fileRecord = fileRecordFactory.build({ name });
+
+				return { fileRecord, name };
+			};
+
+			it('should return the correct name', () => {
+				const { fileRecord, name } = setup();
+
+				const result = fileRecord.getName();
+
+				expect(result).toEqual(name);
+			});
+		});
+	});
+
+	describe('getSchoolId is called', () => {
+		describe('WHEN schoolId exists', () => {
+			const setup = () => {
+				const schoolId = new ObjectId().toHexString();
+				const fileRecord = fileRecordFactory.build({ schoolId });
+
+				return { fileRecord, schoolId };
+			};
+
+			it('should return the correct schoolId', () => {
+				const { fileRecord, schoolId } = setup();
+
+				const result = fileRecord.getSchoolId();
+
+				expect(result).toEqual(schoolId);
+			});
+		});
+	});
+
+	describe('getDescriptions is called', () => {
+		describe('WHEN name, size and mimeType exists', () => {
+			const setup = () => {
+				const name = 'name123';
+				const size = 123;
+				const mimeType = 'mimeType123';
+				const fileRecord = fileRecordFactory.build({ name, size, mimeType });
+
+				return { fileRecord, name, size, mimeType };
+			};
+
+			it('should return a object that include name, size and mimeType', () => {
```

Comment:
Präziser wäre: 'should return an object with name, size and mimeType'

Reply:
done